docs/server/push-duty.rst

```diff
@@ -33,7 +33,7 @@ It's usually the master branch that is tagged::
     $ git checkout master
     $ git pull
     $ git tag 2015.09.10
-    $ git push --tags
+    $ git push 2015.09.10
```

Comment:
I think you need to mention the remote here or it will try and treat `2015.09.10` as the remote. Not great for docs since the remote is probably going to be different for different people but perhaps `upstream` or `mozilla` would be best.

```
> git push 2015.09.10
fatal: '2015.09.10' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.